### PR TITLE
DOC: Fix typos found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
## Summary

Fix typos.

## List of changes proposed in this PR (pull-request)
* Fix the rest of the typos found by [codespell](https://github.com/codespell-project/codespell).
* Remove spaces at end of line.
* Remove empty lines at end of file.
* Run black on files that require it to pass tests.

### CODE REVIEW

**Comment by DimitriPapadopoulos** [comment]
Do I need to run [`black`](https://github.com/psf/black) on modified files?

File: nipype/interfaces/fsl/model_templates/feat_contrastmask_element.tcl [hunk 1]
```diff
@@ -1,3 +1,2 @@
 # Mask real contrast/F-test $c1 with real contrast/F-test $c2?
 set fmri(conmask${c1}_${c2}) 0
-
```
Comment (effigies) [inline nipype/interfaces/fsl/model_templates/feat_contrastmask_element.tcl:3]: I think for template files, we should leave them alone. These are meant to be snippets to compile as inputs to tools, and I don't have an immediate plan to generate before/after files that would use all of these and decide whether the whitespace improved readability or possibly was significant.

**Review by effigies — CHANGES_REQUESTED** [review]
Overall looks good. I think we should revert the `nipype/interfaces/fsl/model_templates/` directory.

File: nipype/interfaces/fsl/model_templates/feat_contrastmask_element.tcl (hunk 1 above)
Comment (DimitriPapadopoulos) [reply nipype/interfaces/fsl/model_templates/feat_contrastmask_element.tcl]: I have reverted `nipype/interfaces/fsl/model_templates/*`.

**Review by effigies — APPROVED** [review]
LGTM!

**Pull request merged**